docs/csharp/programming-guide/nullable-types/handling-null-and-operators.md

```diff
@@ -0,0 +1,80 @@
+# Null coalescing and operators
+
+## Introduction
+
+
+### Null coalescing (??)
+The null coalescing operator consists of a double question mark "??". With this operator, the null check of a reference variable can be simplified greatly.
+
+With the null coalescing operator you can check to see if a variable is null and if it is, return a different value of the same type all in one line. Have a look at the sample below to see how this work.
+
+#### Sample
+In this example we will look at a typical scenario for a null coalescing operator. In the sample code we will see a property with a backing field in a C# class. The first code block shows you how to handle this scenario without the null coalescing operator.
+
+```
+public class NullCoalescingSample1
+{
+	private string _myStringValue;
+	
+	public string MyStringValue
+	{
+		get 
+		{
+			if (_myStringValue == null)
+				return "Default value";
+				
+			return _myStringValue;
+		}
+	}
+}
+```
```

Comment:
I'm also debating if it belongs in a different directory. Let's wait on that for one more iteration.

Reply:
Let's open a PR with the code in the [samples repo](https://github.com/dotnet/samples)  We can include the samples here using the include syntax. (See contributing.md in the root directory)

That way, the sample code is checked with our CI system.

A simple console sample is fine, and use .NET Core since these samples run on all platforms.